components/meter/README.md

```diff
@@ -1,91 +1,138 @@
-# Meter Components
+# Progress Meters
 
-**Note:** these are referred to as "progress meters" on [design.d2l](http://design.d2l/components/progress-meters/).
+Progress meters are used to visually communicate the progress of an object or operation. There are three types of progress meters: Linear, Radial, and Circle. Select the type that best represents your data and fits the visual design.
 
-## d2l-meter-linear
+<!-- docs: demo -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/meter/meter-linear.js';
+  import '@brightspace-ui/core/components/meter/meter-radial.js';
+  import '@brightspace-ui/core/components/meter/meter-circle.js';
+</script>
+<d2l-meter-linear value="30" max="100"></d2l-meter-linear>
+<d2l-meter-radial value="30" max="100"></d2l-meter-radial>
+<d2l-meter-circle value="30" max="100"></d2l-meter-circle>
+```
+
+<!-- docs: start hidden content -->
+### Note: these are referred to as "progress meters" on [design.d2l](http://design.d2l/components/progress-meters/).
+<!-- docs: end hidden content -->
+
+## Best Practices
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Use to show the completion or progress of an object.
+* Use to highlight important or critical progress data.
+* Use to provide feedback when a user completes an action that results in a change in data.
+<!-- docs: end dos -->
+
+<!-- docs: start donts -->
+* Don't use a lot of progress indicators on a single screen because they can overwhelm the user.
+* Don't use motion unless the data is the primary content on the screen.
+<!-- docs: end donts -->
+<!-- docs: end best practices -->
+
+## Light Foreground
```

Comment:
I wasn't 100% sure if this should be left on the page but it seems to fit best as an H2 if we were to leave it. Might be something that gets cut in a design review

Reply:
Actually now that I think about it, this should probably be defined on the mixin as a property. Seems like all of the meter components use this property (and I would think it should show up in the attributes table)

Reply:
So one problem with adding this as an attribute is that we need a dark background to display the light versions properly.  I may leave the attribute undocumented in jsdocs, but add a separate demo that shows the `foreground-light` version. Will need to add some configuration for a dark background in our iframe along with this, maybe a `darkmode:true` or something similar

Reply:
Cool yeah the separate demo probably makes sense for now.